src/utils/manager.py

```diff
@@ -24,9 +25,9 @@
 )
 _mount_parser = re.compile(
```

Comment:
@NucciTheBoss Since this doesn't need to match `nfs` anymore, are there any other reasons to use a regex instead of just calling `split`?

Reply:
Note that the previous regex didn't parse the whole file at once, but had to match every line individually. Performance-wise, I don't think a regex match is faster than just splitting at spaces.